lib/error-handler.ts

```diff
@@ -67,6 +67,11 @@ export default function errorHandler(errorResponse: AxiosError): never {
   const error = new Error()
   error.name =
     errorName && errorName !== 'Unknown' ? errorName : `${response?.status} ${response?.statusText}`
-  error.message = JSON.stringify(errorData, null, '  ')
+
+  try {
```

Comment:
Intentional?

Reply:
yes, as were mentioned in https://github.com/contentful/contentful-management.js/issues/584 if JSON.stringify fails we don't actually know what the real error was at first place

Reply:
Ok, then let's find something that still conveys the minimal amount of information about the error instead of completely silencing it.

Reply:
I updated `catch` branch